tests/test_recorder.py

```
@@ -0,0 +1,40 @@
+                entity_id="sensor.cpap_usage_minutes",
+                platform="resmed_myair",
+                unique_id="resmed_myair_SN123_totalUsage",
+            ),
+            "sensor.other_mask_leak": SimpleNamespace(
+                entity_id="sensor.other_mask_leak",
+                platform="other",
+                unique_id="other_SN123_leakPercentile",
+            ),
+        }
+    )
+    monkeypatch.setattr(recorder.er, "async_get", lambda hass: registry)
```

Ruff (0.15.14)

[warning] Unused lambda argument: `hass`

(ARG005)